core/src/com/unciv/logic/civilization/diplomacy/CityStateFunctions.kt

```diff
@@ -77,7 +77,7 @@ class CityStateFunctions(val civInfo: Civilization) {
 
         fun getVotesFromSpy(spy: Spy?): Float {
             if (spy == null) return 20f
-            var votes = (civInfo.getDiplomacyManager(spy.civInfo)!!.influence / 2)
+            var votes = (civInfo.getDiplomacyManagerOrMeet(spy.civInfo).influence / 2)
```

Comment:
There is a chance apparently, that a civ can have a spy set up in a city-state without seeing them. (From revealing the map with the technology). I think this is because the spy vision isn't updated at the end of the turn to meet the city-state immediately.